akka-docs/rst/scala/typed-actors.rst

```diff
@@ -1,6 +1,13 @@
+.. _typed-actors-scala:
+
 Typed Actors
 ====================
 
+.. note::
+
+  This module is deprecated as it will be superseded by the :ref:`typed-scala`
+  project which is currently being developed in open preview mode.
```

Comment:
I wonder whether a deprecated annotation is already warranted: for someone whose build does not tolerate warnings it would mean that they need to migrate to akka-typed already. My intention is to add this note here as an advanced notice that TypedActors are going away, but we will probably on deprecate them in `(2.4).next`. This should probably be explained in the migration guide, good point.